dask/array/utils.py

```diff
@@ -10,7 +10,6 @@
 import numpy as np
 from tlz import concat, frequencies
 
-from dask.array.core import Array
```

Comment:
nit: I would expect something like core to be importable from pretty much anywhere (same for utils)